services/apps/premium/members_enrichment_worker/src/activities/enrichment.ts

```
@@ -198,6 +214,369 @@
+    if (cache && cache.data) {
+      const normalized = (await normalizeEnrichmentData(
+        source,
+        cache.data,
+      )) as IMemberEnrichmentDataNormalized
+
+      if (Array.isArray(normalized)) {
+        const normalizedContributions = []
+        for (const n of normalized) {
+          if (n.contributions) {
+            normalizedContributions.push(n.contributions)
+            delete n.contributions
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)